sky/cli.py

```diff
@@ -2073,9 +2072,14 @@ def logs(
     assert job_ids is None or len(job_ids) <= 1, job_ids
     job_id = None
     if job_ids:
-        job_id = job_ids[0]
+        if not all(job_id.isdigit() for job_id in job_ids):
+            raise click.UsageError(f'Invalid job ID {", ".join(job_ids)}. '
+                                   'Job ID must be integers.')
+        int_job_ids = [int(job_id) for job_id in job_ids]
+    else:
+        int_job_ids = job_ids
```

Comment:
This naming is a bit confusing as it does not include `None` case. How about just calling it `job_ids_`

Reply:
Changed to `job_ids_` first. How about `job_ids_to_query`..?

Reply:
`job_ids_to_query` sounds good to me.